rlbot/interface.py

```diff
@@ -129,11 +142,9 @@ def stop_match(self, shutdown_server: bool = False):
         flatbuffer = flat.StopCommand(shutdown_server).pack()
         self.send_bytes(flatbuffer, SocketDataType.STOP_COMMAND)
 
-    def start_match(
-        self,
-        match_config: Path | flat.MatchSettings,
-        rlbot_server_port: int = RLBOT_SERVER_PORT,
-    ):
+    def start_match(self, match_config: Path | flat.MatchSettings):
```

Comment:
Does this still work since you removed the call to `run_after_connect`? Basically, does `tests/run_only.py` still work?

Reply:
It does not work since you now must call `ensure_server_started()` first in order to start RLBot. If you don't you will get this behaviour, letting people know they might need to call it:
```
20:47:56  WARNING:   interface[    interface.py:199 -                   connect() ] Connection is being refused/aborted. Trying again ...
20:48:07  WARNING:   interface[    interface.py:199 -                   connect() ] Connection is being refused/aborted. Trying again ...
20:48:26  WARNING:   interface[    interface.py:199 -                   connect() ] Connection is being refused/aborted. Trying again ...
20:49:07  WARNING:   interface[    interface.py:199 -                   connect() ] Connection is being refused/aborted. Trying again ...
Traceback (most recent call last):
  File "C:\Fritid\Code\RLBot\python-interface\tests\run_only.py", line 12, in <module>
    match_manager.start_match(MATCH_CONFIG_PATH, False)
  File "C:\Fritid\Code\RLBot\python-interface\rlbot\managers\match.py", line 239, in start_match
    self.rlbot_interface.connect(
  File "C:\Fritid\Code\RLBot\python-interface\rlbot\interface.py", line 203, in connect
    raise ConnectionRefusedError(
ConnectionRefusedError: Connection was refused/aborted repeatedly! Ensure that Rocket League and the RLBotServer is running. Try calling `ensure_server_started()` before connecting.

Process finished with exit code 1
```

I should add that to the change log and test.